src/toolbar/sticky/stickytoolbarview.js

```diff
@@ -20,21 +20,88 @@ export default class StickyToolbarView extends ToolbarView {
 	/**
 	 * @inheritDoc
```

Comment:
Missing API docs. You added one argument.

Besides, can't we move the limiter to the model?